docs/docs/user/references/components.md

```diff
@@ -210,7 +210,7 @@ Sub class of [KafkaConnector](/docs/user/references/config.md)
 
 ### Usage
 
-Lets other systems push data to Apache Kafka.
+Manages source connectors in your Kafka connect cluster.
```

Comment:
FYI Kafka Connect is not a _cluster_ by itself, rather it is a service that runs on the Kubernetes cluster

Reply:
Connect is a cluster. You can have multiple Connect workers etc. It's a cluster running on a Kubernetes cluster

Reply:
d'oh, my mistake, turns out I actually had no idea haha. please disregard my uneducated comment